h5p-bildetema-words-grid-view/src/components/TopicWordsGrid/TopicWordsGrid.tsx

```diff
@@ -5,14 +5,36 @@ import styles from "./TopicWordsGrid.module.scss";
 
 type TopicWordsGridProps = {
   words: WordType[];
+  showWrittenWords: boolean;
 };
 
-export const TopicWordsGrid: React.FC<TopicWordsGridProps> = ({ words }) => {
+export const TopicWordsGrid: React.FC<TopicWordsGridProps> = ({
+  words,
+  showWrittenWords,
+}) => {
+  const [textVisible, setTextVisible] = React.useState(showWrittenWords);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setTextVisible(e.target.checked);
+  };
+
+  React.useEffect(() => {
+    document
+      .getElementById("toggle")
```

Comment:
This is a little bit scary, because we can't guarantee that no other element has this id. We could perhaps do something with `h5p-bildetema`'s `contentId`.